ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/TransportCallTO.java

```diff
@@ -13,7 +13,7 @@ public record TransportCallTO(
   String universalImportVoyageReference,
   String universalExportVoyageReference,
   PortTerminalLocation location,
-  PortCallStatusCode statusCode,
+  String statusCode,
```

Comment:
Could you add a comment with a link to the possible values?
https://github.com/dcsaorg/DCSA-Information-Model/blob/master/datamodel/referencedata.d/portcallstatuscodes.csv

Reply:
Added the comment with the link and pushed the changes